cmd/osctl/cmd/cluster.go

```diff
@@ -78,11 +78,14 @@ var clusterDownCmd = &cobra.Command{
 func create() (err error) {
 	ctx := context.Background()
 
-	cli, err := client.NewEnvClient()
+	cli, err := client.NewClientWithOpts(client.FromEnv)
 	if err != nil {
 		return err
 	}
 
+	// Ensure docker client version works with server
+	cli.NegotiateAPIVersion(ctx)
+
```

Comment:
This is necessary to bring in the updated docker libs.

`NewEnvClient()` is deprecated in favor of `NewClientWithOpts(...)`
`cli.NegotiateAPIVersion()` handles discovering which version of the api it needs to use to talk with the server.